wireguard-install.sh

```diff
@@ -121,15 +121,15 @@ function installWireGuard() {
 
 	# Install WireGuard tools and module
 	if [[ ${OS} == 'ubuntu' ]]; then
-		apt-get update
-		apt-get install -y wireguard iptables resolvconf qrencode
+		apt-get update && apt-get -y upgrade
+		apt-get install -y linux-headers-$(uname -r) wireguard iptables resolvconf qrencode
```

Comment:
is `linux-headers-$(uname -r)` needed? It should be installed as a dependency if needed

Reply:
> is `linux-headers-$(uname -r)` needed? It should be installed as a dependency if needed

Yes. It is needed to install right kernel headers. 

https://www.scaleway.com/en/docs/installing-wireguard-vpn-linux/#-Installing-and-Configuring-WireGuard-on-the-server

Reply:
The tutorial is out of date, I pinged the team (since I work there).

On ubuntu, the wireguard package is available in the ubuntu repository (https://github.com/angristan/wireguard-install/issues/108) and doesn't require dkms anymore, so no linux-headers. On Debian, we install the package from the backports, so a newer linux version will be installed (and still, headers are not even required).

So in both cases, headers are not used. 🙂

Reply:
> The tutorial is out of date, I pinged the team (since I work there).
> 
> On ubuntu, the wireguard package is available in the ubuntu repository (#108) and doesn't require dkms anymore, so no linux-headers. On Debian, we install the package from the backports, so a newer linux version will be installed (and still, headers are not even required).
> 
> So in both cases, headers are not used. 🙂

As you can see from the 1st message of this PR, on Ubuntu 18.04 wireguard failed to start without proper kernel headers.